config.go

```diff
@@ -87,3 +92,19 @@ func WithBetaVersion(betaVersion ...BetaVersion) ClientOption {
 		c.BetaVersion = betaVersion
 	}
 }
+
+func WithVertexAI(projectID string, location string) ClientOption {
+	return func(c *ClientConfig) {
+		c.BaseURL = fmt.Sprintf("https://%s-aiplatform.googleapis.com/v1/projects/%s/locations/%s/publishers/anthropic/models", location, projectID, location)
```

Comment:
I feel like we might be able to do a better job here using `net/url` to build this url (although I'm sure the `Sprintf` is probably fine for now).

Reply:
I considered using url.JoinPath, but it returns an err.  Since the func has no way to return an err, it would either need to be silently ignored, or panic, neither of which feels like a great option.  However, happy to fix to use this approach.  I did see that client.go line 121 uses Sprintf so it is not unprecedented.